plugins/e621.py

```diff
@@ -86,16 +86,15 @@ def import_submission(self, submission: praw.objects.Submission) -> dict:
                 image_url = url
             else:
                 self.log.debug('Using the e621 API')
-                match_data = match.groupdict()
-                # For non-CDN links, the plugin attempts to get the ID
+                # For non-CDN links, the plugin attempts to get the post_id
                 # out of the URL using regex.
-                id = match_data.get('id')
-                endpoint = 'http://e926.net/post/show.json?id=' + id
-                self.log.debug('Will use API endpoint at ' + endpoint)
+                post_id = match.group('post_id')
+                endpoint = 'http://e926.net/post/show.json?post_id=' + post_id
```

Comment:
Watch out for this: the URL is supposed to use `id`, isn't it? If one intends to change the name of a variable, I'd recommend looking at an IDE/text editor that allows non-destructive renaming of variables, instead of using find-replace.